### DISCUSSION TITLE
Contribution Graph Duplicating Commits

### DISCUSSION THREAD

**Discussion by ghost** [community/community#22542] [New to GitHub]
<p>Some of the commits on my contribution graph are duplicated.</p>

**Comment by jfullstackdev** [comment]
<p>that seems strange, you have blank commits</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/c/4/c4b335dc1f8c509ffc403715b80d68f4b688f3be.png" data-download-href="/uploads/short-url/s45xwrsplKLsGtCGOUc5XNpKaT4.png?dl=1" title="image"><img src="https://user-images.githubusercontent.com/47092464/181071720-67863656-f963-40b5-8381-9fa3190a47ee.png" alt="image" data-base62-sha1="s45xwrsplKLsGtCGOUc5XNpKaT4" width="690" height="373" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/c/4/c4b335dc1f8c509ffc403715b80d68f4b688f3be_2_690x373.png, https://aws1.discourse-cdn.com/github/optimized/3X/c/4/c4b335dc1f8c509ffc403715b80d68f4b688f3be_2_1035x559.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/c/4/c4b335dc1f8c509ffc403715b80d68f4b688f3be_2_1380x746.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/c/4/c4b335dc1f8c509ffc403715b80d68f4b688f3be_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1920×1040 88.2 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>
<p>is this a script ? because if it is not your commits, that’s a security risk, GitHub may notice that,</p>
<p>that’s 22k commits in just one repo and each commit has almost no gap between the other commits,</p>

**Comment by ghost** [comment]
<p>Just was me testing GitHub (this is unrelated to that)</p>

**Comment by ghost** [comment]
<p>My commits in one repo are all duplicated for some reason<br>
I deleted the test repo and the contribution graph still doesn’t work</p>

**Comment by ghost** [comment, marked as answer]
<p>solved, contribution graph works now</p>
